promgen/templates/promgen/service_detail.html

```diff
@@ -8,15 +8,72 @@
 
 {% block content %}
 
-<div class="page-header" v-pre>
-  <h1>Service: {{ service.name }}</h1>
-  {% if service.owner %}
-  <p>{% trans 'Contact' %}: {{service.owner.username}}</p>
-  {% endif %}
+<div class="page-header promgen-flex-space-between-center">
+  <div>
+    <h1>Service: {{ service.name }}</h1>
+    {% if service.owner %}
+    <p>{% translate 'Contact' %}: {{service.owner.username}}</p>
+    {% endif %}
+  </div>
+  
+  <div>
+
+    {% include "promgen/service_action_button.html" %}
+
+    <a @click="setSilenceDataset" data-service="{{service.name}}" class="btn btn-warning btn-sm ml-2 mr-2">{% translate "Silence" %}</a>
+
+    <a href="{% url 'service-update' service.id %}" class="btn btn-warning btn-sm">{% translate "Edit Service" %}</a>
+
+  </div>
 </div>
 
 {% breadcrumb service %}
 
-{% include "promgen/service_block.html" %}
```

Comment:
First, should probably rebase against master.
Second, while this improves the detail page some, it still diverges from the home and service page which I think will cause confusion. We should unify them all at the same time.

`/`
<img width="661" alt="home" src="https://github.com/line/promgen/assets/89725/a749a54b-5cb5-4f4d-aac5-cd3dd03578d5">

`/service/`
<img width="666" alt="service-list" src="https://github.com/line/promgen/assets/89725/2b1d1b71-ba0c-4411-9f56-34fee0ed6be3">

`/service/<id>`
<img width="528" alt="service-detail" src="https://github.com/line/promgen/assets/89725/b21ba98c-8c3e-411a-aa4a-5fa0c5d41d48">


This does *not* mean simply removing `service_block` from here. It means that `service_detail` probably doesn't need many changes, but that many of the navigation changes should be in `service_block` so that it applies to all the pages.

Reply:
em.. I see. You mean each service_block should have that tabs. The style change should adapt to each service_block?
Based on the description of the requirements, I understand that this tab is only needed on the detail page. 
Do we need further discussion about this requirements?

Reply:
The goal is to improve the UI to be easier to understand by users. Having multiple versions of the same controls is probably not moving towards that goal 😅

Reply:
@kfdm I want to confirm what you want. you want to apply same changes about what to show about service on the top page/service list page, correct?

Reply:
If the decision is to move multiple options under an `Actions` dropdown, then we should apply the *same* change to each page that shows a Service block. Having two different ways to display the same buttons adds confusion to users.

Reply:
@kfdm san.  I think there is some misunderstanding here. And I want to confirm again about this PR.
> The goal is to improve the UI to be easier to understand by users. 

First, I completely agree with this, and this is the purpose of this PR.

Second, I want to confirm one thing that.
> This does not mean simply removing service_block from here. It means that service_detail probably doesn't need many changes, but that many of the navigation changes should be in service_block so that it applies to all the pages.

In my PR, to display the content in the original service_block more effectively on the service_detail page, the service_detail has been refactored and no longer simply uses the service_block component.

According to what you mentioned last time, you want the blocks displaying the service details to be consistent on every page where a block with service details is shown.

Does this mean you want the changes made on the service_detail page to also apply to the service_block component, changing the layout of every service_block component to include tabs?


>If the decision is to move multiple options under an Actions dropdown, then we should apply the same change to each page that shows a Service block. Having two different ways to display the same buttons adds confusion to users.

But according to this, it seems like you only want to unify the changes to the Actions dropdown.

So, I am not quite clear about your specific intention. If possible, please explain it to me in more detail.

At last, I would like to share my thoughts. I believe it is acceptable for the list page and the detail page to be different. This is also a common practice in UI design. I don't think this will confuse users; rather, it helps users visually distinguish the functionality of each page. The list page serves as a preview, while the detail page uses tabs to more clearly display specific content.

Reply:
I have to respectfully disagree. Sure there are times where you might use different controls for each page, but I think in this case, there is less confusion by keeping both the same.

To summarize, if the service editing buttons are going to be collapsed to a drop down action button on one page, they should be collapsed to the same for each page they are used. This reduces visual confusion on the frontend, and avoids duplicate code in the html templates.

Reply:
I updated according to the conclusion we made at yesterday's meeting:  should apply the action buttons in service_block.
Please check.